virtool/jobs/build_index.py

```diff
@@ -154,6 +157,59 @@ async def upload(job):
         })
 
 
+async def build_json(job):
+    """
+    Create a reference.json.gz file at ``<data_path>/references/<ref_id>/<index_id>``.
+
+    """
+    await job.db.indexes.find_one_and_update({"_id": job.params["index_id"]}, {
```

Comment:
Move this to when an index document is first created. This is in `virtool.indexes.api`.